examples/rvc-app/rvc-common/pics/RVC_Operational_State_Cluster_Test_Plan.txt

```diff
@@ -0,0 +1,33 @@
+RVCOPSTATE.S=1
+RVCOPSTATE.S.A0000=1
+RVCOPSTATE.S.A0001=1
+RVCOPSTATE.S.A0002=0
+RVCOPSTATE.S.A0003=1
+RVCOPSTATE.S.A0004=1
+RVCOPSTATE.S.A0005=1
+RVCOPSTATE.S.E00=1
+RVCOPSTATE.S.E01=1
+RVCOPSTATE.S.C04.Tx=1
+RVCOPSTATE.S.C00.Rsp=1
+RVCOPSTATE.S.C01.Rsp=0
+RVCOPSTATE.S.C02.Rsp=0
+RVCOPSTATE.S.C03.Rsp=1
+RVCOPSTATE.S.M.ST_STOPPED=1
+RVCOPSTATE.S.M.ST_RUNNING=1
+RVCOPSTATE.S.M.ST_PAUSED=1
+RVCOPSTATE.S.M.ST_ERROR=1
+RVCOPSTATE.S.M.ST_SEEKING_CHARGER=1
+RVCOPSTATE.S.M.ST_CHARGING=1
+RVCOPSTATE.S.M.ST_DOCKED=1
+RVCOPSTATE.S.M.ERR_NO_ERROR=1
+RVCOPSTATE.S.M.ERR_UNABLE_TO_START_OR_RESUME=0
+RVCOPSTATE.S.M.ERR_UNABLE_TO_COMPLETE_OPERATION=0
+RVCOPSTATE.S.M.ERR_COMMAND_INVALID_IN_STATE=1
```

Comment:
Why are ERR_UNABLE_TO_START_OR_RESUME and ERR_UNABLE_TO_COMPLETE_OPERATION zero? I thought that you could actually trigger those.

Reply:
I must have overlooked them while using the UI. I have fixed this [here](https://github.com/project-chip/connectedhomeip/pull/29159/commits/9f0e5bd6186da34c985fe079a32ca35506ad8744).